pkg/agent/cniserver/pod_configuration.go

```diff
@@ -237,33 +314,40 @@ func configureInterface(
 	// create OVS Port and add attach container configuration into external_ids
 	ovsPortName := hostIface.Name
 	klog.V(2).Infof("Adding OVS port %s for container %s", ovsPortName, containerID)
-	portUUID, err := setupContainerOVSPort(ovsBridge, containerConfig, ovsPortName)
+	portUUID, err := pc.setupContainerOVSPort(containerConfig, ovsPortName)
 	if err != nil {
 		return err
 	}
 
 	// Rollback to remove OVS port if hit error in later manipulations
 	defer func() {
 		if !success {
-			ovsBridge.DeletePort(portUUID)
+			pc.ovsBridgeClient.DeletePort(portUUID)
 		}
 	}()
 
 	// GetOFPort will wait for up to 1 second for OVSDB to report the OFPort number.
-	ofPort, err := ovsBridge.GetOFPort(ovsPortName)
+	ofPort, err := pc.ovsBridgeClient.GetOFPort(ovsPortName)
 	if err != nil {
 		klog.Errorf("Failed to get of_port of OVS interface %s: %v", ovsPortName, err)
 		return err
 	}
-	klog.V(2).Infof("Setting up Openflow entries for container %s", containerID)
-	err = ofClient.InstallPodFlows(ovsPortName, containerConfig.IP, containerConfig.MAC, gateway.MAC, uint32(ofPort))
+	// Setup openflow entries for OVS interface
+	klog.V(2).Infof("Setting up openflow entries for container %s", containerID)
```

Comment:
Do we want to use **O**penflow for consistency?

Reply:
Sure.